src/start-sc.ts

```diff
@@ -1,17 +1,17 @@
-import {debug, getInput, isDebug, warning} from '@actions/core'
+import {debug, getInput, isDebug, warning, error} from '@actions/core'
 import {which} from '@actions/io'
 import {spawn} from 'child_process'
 import {info} from 'console'
-import {mkdtempSync, readFileSync} from 'fs'
+import {mkdtempSync, readFileSync, existsSync, openSync} from 'fs'
 import {tmpdir} from 'os'
 import {dirname, join} from 'path'
 import optionMappingJson from './option-mapping.json'
 import {stopSc} from './stop-sc'
 import {wait} from './wait'
 
-const tmp = mkdtempSync(join(tmpdir(), `sauce-connect-action`))
-const LOG_FILE = join(tmp, 'sauce-connect.log')
-const READY_FILE = join(tmp, 'sc.ready')
+let tmp = '';
+let LOG_FILE = '';
+let READY_FILE = '';
```

Comment:
These three work better as constants, since they don't actually change. They could be renamed `DEFAULT_LOG_FILE`, etc. for clarity. Also, `tmp` -> `TMP_FOLDER` if it's used as a global variable in `startSc`